test/taxon.jl

```diff
@@ -0,0 +1,18 @@
+module TestSpecies
+
+  using GBIF
+  using Test
+
+  iver = taxon("Iris versicolor", rank=:SPECIES)
```

Comment:
I wonder if `rank` might be an enum, that would enforce certain values?

Reply:
It is checked internally, but an enum is also a good solution, will do.